packages/wrangler/src/pages/dev.ts

```diff
@@ -545,8 +567,10 @@ export const Handler = async ({
 				};
 			})
 			.filter(Boolean) as AdditionalDevProps["durableObjects"],
-		r2: r2s.map((binding) => {
-			return { binding: binding.toString(), bucket_name: binding.toString() };
+		r2: r2s.map((r2) => {
+			const { binding, ref } = BINDING_REGEXP.exec(r2.toString())?.groups || {};
```

Comment:
Under what circumstance would the regex fail and return `{}`? Could that happen unexpectedly?